android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -801,10 +801,23 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
 
     @Override
     public void onReceivedSslError(final WebView webView, final SslErrorHandler handler, final SslError error) {
+        // WebView.getUrl() will return the top-level window URL.
```

Comment:
Perhaps this comment could open with the simple observation that `onReceivedSslError` is called for subresources in addition to the top-level URL? That's key info that, like you, I had not expected (given that many other Android WebView events are not raised for subresources).

Reply:
Will update shortly.